resources/usr/local/etc/sphinx/configs-available/everything.sphinx.conf

```diff
@@ -0,0 +1,720 @@
+  sql_range_step = 10000
+  sql_query_pre = set names utf8
+  sql_query_pre = replace into GDN_SphinxCounter select 4, max(GroupID) from GDN_Group
+  sql_query = select \
+        GroupID * 10 + 4, \
+        GroupID, \
+        3000000000 + GroupID as DiscussionID, \
+        0 as CategoryID, \
+        InsertUserID, \
+        unix_timestamp(DateInserted) as DateInserted, \
+        Name, \
+        Description as Body, \
+        400 as dtype, \
+        0 as IP, \
+        1000 as Score \
+    from GDN_Group \
+    where \
+       GroupID >= $start AND \
+       GroupID <= $end
+  sql_attr_uint = GroupID
+  sql_attr_uint = DiscussionID
+  sql_attr_uint = CategoryID
+  sql_attr_uint = InsertUserID
+  sql_attr_timestamp = DateInserted
+  sql_attr_float = Score
+  sql_attr_uint = dtype
+  sql_attr_uint = IP
+}
+
+source vanilla_dev_Group_Delta: vanilla_dev_Group {
+  sql_query_range = SELECT (select MaxID from GDN_SphinxCounter where CounterID = 4), MAX(GroupID) FROM GDN_Group
+  sql_query_pre = set names utf8
+  sql_query = select \
+       GroupID * 10 + 4, \
+       GroupID, \
+       3000000000 + GroupID as DiscussionID, \
+       0 as CategoryID, \
+       InsertUserID, \
+       unix_timestamp(DateInserted) as DateInserted, \
+       Name, \
+       Description as Body, \
+       400 as dtype, \
+       0 as IP, \
+       1000 as Score \
+    from GDN_Group \
+    where \
+       GroupID >= $start AND \
+       GroupID <= $end
+}
+
+index vanilla_dev_Group {
+  source = vanilla_dev_Group
+  path = /usr/local/etc/sphinx/data/vanilla_dev_Group
+  morphology = stem_en
+  stopwords = /usr/local/etc/sphinx/data/stops.txt
+}
+
+index vanilla_dev_Group_Delta: vanilla_dev_Group {
+  source = vanilla_dev_Group_Delta
+  path = /usr/local/etc/sphinx/data/vanilla_dev_Group_Delta
+}
+
+source vanilla_dev2_Group {
+  type = mysql
+  sql_host = database
+  sql_user = root
+  sql_pass =
+  sql_db = vanilla_dev2
+  sql_query_range = SELECT MIN(GroupID),(select MaxID from GDN_SphinxCounter where CounterID = 4) FROM GDN_Group
+  sql_range_step = 10000
+  sql_query_pre = set names utf8
+  sql_query_pre = replace into GDN_SphinxCounter select 4, max(GroupID) from GDN_Group
+  sql_query = select \
+        GroupID * 10 + 4, \
+        GroupID, \
+        3000000000 + GroupID as DiscussionID, \
+        0 as CategoryID, \
+        InsertUserID, \
+        unix_timestamp(DateInserted) as DateInserted, \
+        Name, \
+        Description as Body, \
+        400 as dtype, \
+        0 as IP, \
+        1000 as Score \
+    from GDN_Group \
+    where \
+       GroupID >= $start AND \
+       GroupID <= $end
+  sql_attr_uint = GroupID
+  sql_attr_uint = DiscussionID
+  sql_attr_uint = CategoryID
+  sql_attr_uint = InsertUserID
+  sql_attr_timestamp = DateInserted
+  sql_attr_float = Score
+  sql_attr_uint = dtype
+  sql_attr_uint = IP
+}
+
+source vanilla_dev2_Group_Delta: vanilla_dev2_Group {
```

Comment:
Sorry. That is my garbage here. We need to remove  following three sections with `vanilla_dev2`

Reply:
https://github.com/vanilla/vanilla-docker/pull/62